Documentation/Hosting/AmazonGameLift/GameLift.md

```diff
@@ -0,0 +1,105 @@
+# MultiplayerSample Project with Amazon GameLift
+
+This README covers testing and running MultiplayerSample with Amazon GameLift.
+
+## Running with [Amazon GameLift](https://docs.aws.amazon.com/gamelift/index.html)
+
+### Setup
+1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to interact with Amazon GameLift.
+1. Enable the "AWSGameLift" and "MPSGameLift" gem by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
+1. Build the server, game launchers, and asset bundler for MultiplayerSample as normal, per [top-level README](/README.md).
+    ```sh
+    cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m /nologo
+    ```
+1. Build all the assets
+    ```sh
+    cmake --build build\windows --target MultiplayerSample.Assets --config profile -- /m /nologo
+    ```
+1. Work in progress (WiP) step: Add your AWS region to Config/default_aws_resource_mappings.json (example: "Region": "us-west-2")
+    a. Currently needed otherwise when the client initializes GameLift there will be an error about not having a region. 
+    b. This step will be removed once we properly parse the game-session data which contains the fleet-id, region-id, etc  
+
+## Build Server for Windows (WiP)
```

Comment:
is this no longer WIP?

Reply:
Still WiP. "Configure" requires a region, but I havent investigated why.